### ISSUE TITLE
Not able to execute `use [database]` statement

### ISSUE THREAD

**Issue by embojker** [dbeaver/dbeaver#24128]
### Description

Not able to execute a SQL file which affects multiple databases and so uses `use [database_name]` statements throughout the script. 

The error message
```
Error occurred during SQL script execution

Reason:
SQL Error [1064] [42000]: (conn=10) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'use myDatabase' at line 5
```

Downgrading to version 24.0.0 seems to have solved the problem.

### DBeaver Version

Community Edition 24.0.3

### Operating System

Windows 11 Pro

### Database and driver

Ver 15.1 Distrib 10.4.22-MariaDB, for Win64 (AMD64), source revision a19ab67318760f8f155ef7f4f821dfc738542c67

### Steps to reproduce

Execute a script that has the following structure:

```
use myDatabase1;
-- Do something with the myDatabase1, e.g. update some rows.

use myDatabase2;
-- Do something with the myDatabase2, e.g. update some rows.

use myDatabase3;
-- Do something with the myDatabase3, e.g. update some rows.
```

The script would (error) stop at line `use myDatabase2;` with the error message:
```
Error occurred during SQL script execution

Reason:
SQL Error [1064] [42000]: (conn=10) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'use [database_name]' at line 5
```

**Expected behavior**
The script should execute as before, without errors.

**Comment by mattstrain** [comment]
So i had this error today too (SQL Error [1064] [42000]), for a database upgrade script for an up coming app release.

The script was working fine before upgrading DBeaver this morning (apologies I don't know what version i upgraded from, it was maybe a couple of months ago i did an upgrade). However after the upgrade to version 24.0.3.202404211624 my sql script was failing all over the place but the same statements would run fine when run individually in a separate window.
 
 it seems to be caused by the setting in:
Settings -> Editors -> SQL Editor -> SQL Processing -> "Blank Line is a statement delimiter" -> Smart

I set this to never and it now runs fine. 

It seems DBeaver is ignoring semi colon delimiter's at the end of a statement and wants blank lines instead?  Note I was hitting the execute script option to run all statements and each statement is delimited with a ;

**Comment by E1izabeth** [comment]
Thank you for bug report

**Comment by E1izabeth** [comment]
The fix will be included in DBeaver 24.0.4.
Also you can try Early Access version with fix https://dbeaver.io/files/ea/ now.
